PyInstaller/hooks/graph/hook-gi.repository.GLib.py

```diff
@@ -0,0 +1,5 @@
+from PyInstaller.lib.modulegraph.modulegraph import RuntimeModule
+
+
+def hook(module_graph):
+    module_graph.add_module(RuntimeModule('gi.repository.GLib'))
```

Comment:
Why is this necessatry. Why doesn't just adding `hiddenimports = ['gi.repository.GLib']` work? Please put the reason into the hook as comment.
(This applies to all hooks here.)